### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VisaBuy**

### CODE REVIEW

**Review by zwavy310 — APPROVED** [review]

**Review by Kampfkarren — CHANGES_REQUESTED** [review]
Hmm

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Third-Party Libraries
 
 This repository contains certain open source third-party software used by the Unreal Engine.
+Draze BRN
```
Comment (chesse20) [inline README.md:4]: wow

**Review by chesse20 — APPROVED** [review]
yea

File: README.md (hunk 1 above)
Comment (billygoatj) [reply README.md:4]: Looks good

**Review by Kpryz75 — APPROVED** [review]

**Review by Kenner-G — CHANGES_REQUESTED** [review]
[SPAM]

Do not approve.

**Review by RDIL — CHANGES_REQUESTED** [review]
nah

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+HydreKayzi 
+DrazeXMK
```
Comment (chesse20) [inline README.md:6]: add my name here to

**Review by chesse20 — CHANGES_REQUESTED** [review]

**Review by jondamico1 — APPROVED** [review]

**Review by stephensjames177 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by jgreever — CHANGES_REQUESTED** [review]
[SPAM]

This will be rejected as no needed/useful changes were done. Adding names for the sake of it ruins the idea of this being an open organization. Time to close the doors to the public if this behavior keeps up.

Add useful/needed changes only, please do not add spam to this repo like this again.

File: README.md (hunk 2 above)
Comment (ni3nh) [reply README.md:6]: n

**Comment by Kampfkarren** [comment]
[SPAM]

This will be rejected as no needed/useful changes were done. Adding names for the sake of it ruins the idea of this being an open organization. Time to close the doors to the public if this behavior keeps up.

Add useful/needed changes only, please do not add spam to this repo like this again.

**Comment by bl1mp** [comment]
> [SPAM]
> 
> This will be rejected as no needed/useful changes were done. Adding names for the sake of it ruins the idea of this being an open organization. Time to close the doors to the public if this behavior keeps up.
> 
> Add useful/needed changes only, please do not add spam to this repo like this again.

agreed.

**Comment by chesse20** [comment]
> 
> 
> > [SPAM]
> > This will be rejected as no needed/useful changes were done. Adding names for the sake of it ruins the idea of this being an open organization. Time to close the doors to the public if this behavior keeps up.
> > Add useful/needed changes only, please do not add spam to this repo like this again.
> 
> agreed.

What

File: README.md (hunk 1 above)
Comment (Messhias) [inline README.md:4]: Remove names, this isn't useful for organizations and open source. Please be aware to follow the culture.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
 # Third-Party Libraries
 
 This repository contains certain open source third-party software used by the Unreal Engine.
+Draze BRN
+HydreKayzi
```
Comment (Messhias) [inline README.md:5]: Remove names, this isn't useful for organizations and open source. Please be aware to follow the culture.

File: README.md (hunk 2 above)
Comment (Messhias) [inline README.md:6]: Remove names, this isn't useful for organizations and open source. Please be aware to follow the culture.

**Review by Messhias — CHANGES_REQUESTED** [review]
Please remove your names, there's no reason to open a PR just to add your names, what's the use this for open source software?

**Comment by chesse20** [comment]
can they add my name too

**Comment by Messhias** [comment]
No.

**Pull request closed without merging**